### PR TITLE
Add overflow hint for text

### PR DESCRIPTION
**Pull request by filtsin**
Add overflow hint "..." for very big text in list_items and input.

screen

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -47,6 +47,7 @@ syslog = "6.0.0"
 defaults = "0.2.0"
 freedesktop-icon-lookup = "0.1.0"
 sublime_fuzzy = "0.7.0"
+keyframe = "1.1.1"
```
Comment (l4l) [inline Cargo.toml:50]: Looks unused

**Review by l4l — APPROVED** [review]

File: Cargo.toml (hunk 1 above)
Comment (filtsin) [reply Cargo.toml]: Removed

Comment (l4l) [reply Cargo.toml]: And cargo.lock :) apparently need to check that in ci

Comment (filtsin) [reply Cargo.toml]: Oops. Sorry about that

**Pull request merged**